.github/workflows/terraform-plan-apply.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: |
           git fetch origin main:localmain
 
-          export FILES=$(git diff --name-only localmain | grep .json)
+          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
```

Comment:
can you add a comment that explains what this does?
like:  gets changed json files... without the schema one...

Reply:
Added the comment